main/res/values/strings.xml

```diff
@@ -812,11 +812,9 @@
     <string name="map_source_osm_osmde">OpenStreetMap.de</string>
     <string name="map_source_osm_cyclosm">CyclOSM</string>
     <string name="map_source_osm_offline">Offline</string>
-    <string name="map_source_osm_offline_attribution_pleasewait">(Please wait…)</string>
+    <string name="map_source_osm_offline_attribution_pleasewait">Please wait…</string>
     <string name="map_source_osm_offline_combined">Combined (Offline)</string>
-    <string name="map_source_osm_offline_combined_attribution_pleasewait">Combined (%d maps, please wait…)</string>
-    <string name="map_source_osm_offline_combined_attribution_details">Combined (%d maps, %d attributions): click for details</string>
-    <string name="map_source_osm_offline_combined_attribution_single">Combined (%d maps): %s</string>
+    <string name="map_source_osm_offline_combined_attribution_details">%d attributions, click for details</string>
```

Comment:
This is a `PluralsCandidate` for the lint check, see e.g.
https://ci.cgeo.org/job/cgeo%20pull%20request/3919/androidLintResult/package.-963969145/source.-6884541216426645145/#819
Please have a look at the discussion in https://github.com/cgeo/cgeo/issues/8456

Reply:
> This is a `PluralsCandidate` for the lint check, see e.g.
> https://ci.cgeo.org/job/cgeo%20pull%20request/3919/androidLintResult/package.-963969145/source.-6884541216426645145/#819
> Please have a look at the discussion in https://github.com/cgeo/cgeo/issues/8456

This translation is used in code only if given parameter is 2 or more (never for singular case), see other parts of this PR. Thus the singular/plural problem does not apply here imho.

Reply:
Please see at https://github.com/cgeo/cgeo/issues/8456#issuecomment-645583358.
`And the rules for those numbers seem to vary a lot from language to language, according to the docs we need to support zero, one, two, few, many and other, although most languages only use one or two of those identifiers.`
So we have to support two, few, many and other here.

Reply:
Strings are irrelevant with newest changes, so I guess discussion can be resolved